### ISSUE TITLE
Fix the Footer Section

### ISSUE THREAD

**Issue by T1rth1** [mindspace-prototype/MindSpace#158]
By making the About Us more left align and make Developers  photos in more good manner(small or medium size) and align by row then it look more clear and neat..so i would like to work upon this issue..please assign me this issue

**Comment by 761jayanth8** [comment]
Yes Sir I completely understand your problem statement and I have fixed this one Sir and here are the images of both small and medium devices about a section sir if you like it please make me as contributor.
![Screenshot 2024-05-11 021859](https://github.com/The-MindSpace/MindSpace-Web/assets/136685608/f7b8d935-f099-4735-9f4a-96df5885dd6a)
![Screenshot 2024-05-11 021917](https://github.com/The-MindSpace/MindSpace-Web/assets/136685608/e531e9bf-d313-47e7-80b5-10861b23b1bd)

**Comment by Ankit389** [comment]
Please assign me this issue #158

**Comment by komalgururanii** [comment]
hey as a Web Developer i would like to fix your footer section and make more reliable and attractive and functional. I would be grateful if you could consider assigning me the footer task. Please let me know if there are any specific requirements or guidelines I should follow.

Thank you for considering my request. I look forward to your response.

**Comment by laxmithapas** [comment]
Hi admin, I would like to work on this footer section problem, please assign it to me.
Thanks!'
